luxonis_train/nodes/ocr_decoder.py

```diff
@@ -0,0 +1,171 @@
+"""ResNet backbone.
+
+Source: U{https://github.com/hailo-ai/LPRNet_Pytorch/blob/master/model/LPRNet.py}
+@license: U{PyTorch<https://github.com/hailo-ai/LPRNet_Pytorch?tab=Apache-2.0-1-ov-file#readme>}
+"""
+from typing import Literal
+
+import torch
+import torch.nn as nn
+import torchvision
+from torch import Tensor
+
+from .base_node import BaseNode
+from luxonis_train.utils.types import LabelType
+
+
+class ResBlock(nn.Module):
```

Comment:
I believe both `ResBlock` nad `UpSample` should already be implemented in `luxonis_train.nodes.blocks` or in one of the backbones